rotkehlchen/tests/integration/test_premium.py

```diff
@@ -218,6 +218,9 @@ def test_try_premium_at_start_new_account_pull_old_data(
 
     For a new account
     """
+    with open(Path(__file__).resolve().parent.parent / 'data' / 'remote_old_db.txt', 'rb') as f:
```

Comment:
I am bothered by the names of those files. Can you name them something like `remote_old_encrypted_db.txt`

Other file same too.

```suggestion
    with open(Path(__file__).resolve().parent.parent / 'data' / 'remote_old_db.txt', 'rb') as f:
```